perception/object_recognition/detection/roi_cluster_fusion/include/roi_cluster_fusion/roi_cluster_fusion_nodelet.hpp

```diff
@@ -11,67 +11,99 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#pragma once
-#include "rclcpp/rclcpp.hpp"
-#include "tf2_ros/buffer.h"
-#include "tf2_ros/transform_listener.h"
+
+#ifndef ROI_CLUSTER_FUSION__ROI_CLUSTER_FUSION_NODELET_HPP_
+#define ROI_CLUSTER_FUSION__ROI_CLUSTER_FUSION_NODELET_HPP_
+
+#include <boost/circular_buffer.hpp>
+#include <map>
 #include <memory>
+#include <vector>
+
 #include "autoware_perception_msgs/msg/dynamic_object_with_feature.hpp"
 #include "autoware_perception_msgs/msg/dynamic_object_with_feature_array.hpp"
+#include "cv_bridge/cv_bridge.h"
+#include "image_transport/image_transport.h"
```

Comment:
replace image_transport.h with image_transport.hpp.
`   41 | #pragma message ("Warning: This header is deprecated. Use 'image_transport.hpp' instead")`

Reply:
warning occurs but there is no such mage_transport.hpp.

Reply:
fixed at
5c6743e

Reply:
Could you check if image_transport.hpp exists?
```
❯ fdfind image_transport.hpp /opt/ros/foxy/
/opt/ros/foxy/include/image_transport/image_transport.hpp
```

Reply:
somewhat I couldnt install image transport at that time but I checked now
There was /opt/ros/foxy/include/image_transport/image_transport.hpp
so fixed 9d37541